webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/resource/visualizer_ui.js

```diff
@@ -553,6 +554,50 @@ var VisualizerUI = (function($, window, undefined) {
 
             /* END comment popup - related */
 
+// BEGIN WEBANNO EXTENSION - #1697 - Explicit UI for accepting/recejcting recommendations
+            var displayButtonsTimer = null;
+            var buttonsShown = false;
+            var displaySpanButtons = function (evt, target, spanId) {
+                if(target.attr('data-span-id')) {
+                    var spanPosition = target.position();
+                    var spanWidth = target.width();
+                    var spanHeight = target.height();
+                    var acceptBtn = $('<button id="span_accept_btn">Accept</button>')
```

Comment:
Optimally, we shouldn't use fixed IDs here. I don't see references to `span_accept_btn` and `span_reject_btn` elsewhere, so I guess these can be removed?